exercises/concept/tim-from-marketing-2/src/test/java/EmployeeDatabaseTest.java

```diff
@@ -0,0 +1,102 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Optional;
+
+public class EmployeeDatabaseTest {
+
+    private EmployeeDatabase employeeDatabase;
+    private List<Employee> listOfEmployees = new ArrayList<>();
+
+    void initList() {
+        listOfEmployees.add(new Employee(1, "Tim", "Marketing"));
+        listOfEmployees.add(new Employee(2, "Joe", "Sales"));
+        listOfEmployees.add(new Employee(3, "Jane", "IT"));
+        listOfEmployees.add(new Employee(4, null, null));
+        listOfEmployees.add(new Employee(5, null, "IT"));
+        listOfEmployees.add(new Employee(6, "Alice", null));
+    }
+
+    @BeforeEach
+    void setup() {
+        initList();
+        employeeDatabase = new EmployeeDatabase(listOfEmployees);
+    }
+
+    @Test
+    @Tag("task:1")
+    @DisplayName("Retrieve one employee by id")
+    void retrieveOneEmployeeById() {
+        assertThat(employeeDatabase.getEmployeeById(2))
+                .isEqualTo(Optional.of(listOfEmployees.get(1)));
+    }
+
+    @Test
+    @Tag("task:2")
+    @DisplayName("Retrieve other employee by id")
+    void retrieveOtherEmployeeById() {
+        assertThat(employeeDatabase.getEmployeeById(3))
+                .isEqualTo(Optional.of(listOfEmployees.get(2)));
+    }
+
+    @Test
+    @Tag("task:3")
```

Comment:
There are only two tasks in the instructions, so the tags should be only `"task:1"` or `"task:2"`. They should line up with the corresponding task in the instructions.